client/src/App.jsx

```diff
@@ -43,7 +43,6 @@ function App() {
         <BrowserRouter>
           {/* Landing Page */}
           <Route exact path="/">
-            <NavBar />
```

Comment:
Is this now being rendered in `<Landing>` or did you decide to change the layout?

Reply:
@jaygraygray I changed the layout of the landing page.